habit/src/containers/login/LoginScreen.js

```diff
@@ -2,18 +2,44 @@ import React from 'react';
 import { View } from 'react-native'
 import CustomTextInput from '../../components/TextInput'
 import * as CONSTANTS from '../../utils/string_constants'
+import {Button} from 'react-native-elements'
+import styles from '../../styles/login'
 
-import styles from '../../styles/common'
+export default class Login extends React.Component{
+    constructor(props){
+        super(props);
+        this.state = {
+            firstname: '',
+            lastname: '',
+            password: ''
+        }
+    }
 
-export default function Login(props){
-    let [input, setInput] = React.useState('')
-    return(
-        <View style={styles.page}>
-            <CustomTextInput 
-                placeholder = {CONSTANTS.USERNAME}
-                value = {input}
-                onChangeText = { (value) => setInput(value) }
-            />
-        </View>
-    )
+    handleLogin(){
+        //add logic
+    }
+
+    render(){
+        return(
+            <View style={styles.loginContainer}>
+                <CustomTextInput 
+                    placeholder = {CONSTANTS.USERNAME}
+                    value = {this.state.firstname}
+                    onChangeText = { value => this.setState({firstname: value}) }
+                />
+                <CustomTextInput 
+                    placeholder = {CONSTANTS.PASSWORD}
+                    value = {this.state.password}
+                    secureTextEntry={true}
+                    onChangeText = { value => this.setState({password: value}) }
+                />
+                <Button
+                    title="Login"
+                    type="clear"
```

Comment:
What does this do? 🤨

Reply:
this?

Reply:
clear? it's a style provided by react native elements